Makefile

```diff
@@ -121,6 +121,7 @@ client-python: api/swagger.yml  ## Generate SDK for Python client
 		-o /mnt/clients/python
 
 client-java: api/swagger.yml  ## Generate SDK for Java (and Scala) client
+	find clients/java -mindepth 1 -delete
```

Comment:
This does not seem a safe thing to do.  If we care about specific names, let's delete only them.  Otherwise I'd be happiest to delete _everything_.  E.g. if everything were generated into a single directory that *only* contained generated code, we could wipe that directory and always be safe.